proto/proto/requests.proto

```diff
@@ -4,18 +4,24 @@ package requests;
 import "account_id.proto";
 import "block_header.proto";
 import "digest.proto";
-import "note.proto";
 
 message AccountUpdate {
     account_id.AccountId account_id = 1;
     digest.Digest account_hash = 2;
 }
 
+message NoteCreated {
+    digest.Digest note_hash = 3;
+    uint64 sender  = 4;
+    uint64 tag = 5;
+    uint32 num_assets = 6;
+}
```

Comment:
nit: can we use `NoteEnvelope`, and `NoteMetada` as we use in `miden-base`, so that we reduce the number of names in our system?

Reply:
I wonder if we need the `NoteEnvelope` and `NoteMetadata` structs to being with. Specially without any serialization, since that is clearly intended to be used in the rollup which by definition requires networking.

Anyways, changed it.

Reply:
reverted it because we need the note position